eth2deposit/cli/generate_keys.py

```diff
@@ -39,23 +41,23 @@ def validate_password(cts: click.Context, param: Any, password: str) -> str:
     try:
         validate_password_strength(password)
     except ValidationError as e:
-        click.echo(f'Error: {e} Please retype.')
+        click.echo(e)
     else:
         is_valid_password = True
 
     while not is_valid_password:
-        password = get_password(text='Type the password that secures your validator keystore(s)')
+        password = get_password(load_text(['msg_password_prompt']))
         try:
             validate_password_strength(password)
         except ValidationError as e:
-            click.echo(f'Error: {e} Please retype.')
+            click.echo(e)
         else:
             # Confirm password
-            password_confirmation = get_password(text='Repeat for confirmation')
+            password_confirmation = get_password(load_text(['msg_password_confirm']))
             if password == password_confirmation:
                 is_valid_password = True
             else:
-                click.echo('Error: the two entered values do not match. Please retype again.')
+                click.echo(load_text(['err_password_mismatch']))
 
     return password
 
```

Comment:
(couldn't comment on L63 😅)

Some messages in the following `validate_eth1_withdrawal_address` also need to be added to JSON file and translated:

L67:  `raise ValueError("The given Eth1 address is not in hexadecimal encoded form.")`
and
L70: `click.echo(f'\n**[Warning] you are setting Eth1 address {normalized_address} as your withdrawal address. Please ensure that you have control over this address.**\n')`

Reply:
Resolved in https://github.com/ethereum/eth2.0-deposit-cli/pull/182/commits/06d79203d4d11b097fbfc5a2dafe89dc90ae97ab